src/modules/storage/LedgerStorage.ts

```diff
@@ -1806,4 +1806,28 @@ export class LedgerStorage extends Storages
             WHERE height = ${cur_height};`;
         return this.query(sql, []);
     }
+
+    /**
+     * Gets block height and merkle root
+     * @param tx_hash The hash of the transaction
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the records
+     * and if an error occurs the `.catch` is called with an error.
+     */
+    public getBlockHeaderByTxHash (tx_hash: Hash): Promise<any[]>
+    {
+        let sql =
+        `SELECT
+            height, merkle_root
```

Comment:
How about including all the block header information?

Reply:
Yes, since the name is `getBlockHeaderByTxHash`.
@MichaelKim20 what do you think?

Reply:
Okay 
Let's add what we need next time.